src/encoded/static/components/search.js

```diff
@@ -509,22 +508,16 @@ var ResultTable = search.ResultTable = React.createClass({
         }
         // Check to see if we are searching among multiple data types
         // True if only facet is of field "type" when ignoring audits
-        var facet_types = [];
         for (var i = 0; i < facets.length; i++){
-            if (facets[i]['field']){
-                if (!(facets[i]['field'].includes("audit"))){
-                    facet_types.push(facets[i]['field'])
+            if (facets[i]['field'] && facets[i]['field'] == 'type'){
```

Comment:
it is worth breaking this out into a function called get_facet_filter perhaps just for readability?